tasks/lib/index.js

```diff
@@ -121,10 +124,8 @@ var createAndUploadArtifacts = function (options, done) {
 
     artifactStream.on('data', function(chunk) {
 
-        var binaryChunk = chunk.toString('binary');

```

Comment:
Alternatively this binary conversion could be activated not on Windows only. Just query same variable if on Windows platform and then activate it if not Windows.

What do you think about that ?